payload-files/hapi/_test-param-error_{param}--event.json

```diff
@@ -13,7 +13,7 @@
       "sample_rate": "1",
       "sampled": "true",
       "trace_id": "[[ID2]]",
-      "transaction": "/test-param-error/{param}"
+      "transaction": "GET /test-param-error/1337"
```

Comment:
The error event does not get parameterized in rc.1. @mydea

Reply:
hmm that is... not ideal! Could you look into this? Also cc @Lms24 (when you're back from vacation) as you worked on these things before - ideally we can figure this out before v8 stable goes out, but other than this should not block us either!